### ISSUE TITLE
Empty response from API

### ISSUE THREAD

**Issue by AngelsDustz** [Montapacking/magento-monta-checkout#34]
When implementing this module for a customer we noticed no results from the API.
Upon investigation it turns out that the https://github.com/Montapacking/magento-monta-checkout/blob/6cdfd57932d23f46b0325d9858fbf31e3a280aba/Api/MontapackingShipping.php#L344 getShippingOptions is used but doesn't return anything. Whereas the getShippers works but breaks on a undefined class MontaCheckout_Shipper https://github.com/Montapacking/magento-monta-checkout/blob/6cdfd57932d23f46b0325d9858fbf31e3a280aba/Api/MontapackingShipping.php#L230

The pickup points also returns an empty array. When checking the logs we can see the following:

```
MontapackingLogger.NOTICE: Connection logged (https://api.montapacking.nl/rest/v5/shippingoptions?&Origin=<redacted>&Currency=EUR&Language=NL&OnlyPickupPoints=true&ProductsOnStock=TRUE&MaiboxShipperMandatory=0&TrackingMandatory=0&InsuranceRequired=0&ShipmentFitsThroughDutchMailbox=0&OrderValueInclVat=66.0700&OrderValueExclVat=66.0700&Address.Street=Hogeweg&Address.HouseNumber=20&Address.HouseNumberAddition=&Address.PostalCode=9472pb&Address.City=Zuidlaren&Address.State=&Address.CountryCode=NL&Address.Latitude=53.0751003&Address.Longitude=6.698484&0=PAK&1=DHLservicepunt&2=DPDparcelstore&3=AFH&4=DHLParcelConnectPickupPoint) {"source":"Montapacking Checkout"} []

```

But no results.

**Comment by jvdsmonta** [comment]
Shipping options have to be enabled in the backend (Montaportal) before they are returned by the API. Most probably no Shipping options are enabled.

**Comment by AngelsDustz** [comment]
Ah, I forgot to mention when we do the request manually or call the `_call` function manually we get results. I will double check the back-end.

**Comment by andrevlot** [comment]
Dear AngelsDustz,

Did you check this already? I also think there were no shippers available.

**Comment by AngelsDustz** [comment]
Yes, I did check it with a good API that has shippers available as in #32 however no results.

**Comment by AngelsDustz** [comment]
A new API key was created and that seems to have resolved the issue.
